sbin/jtreg.sh

```diff
@@ -22,12 +26,20 @@ JTREG_VERSION=${JTREG_VERSION:-4.2.0-tip}
 JTREG_TARGET_FOLDER=${JTREG_TARGET_FOLDER:-jtreg}
 JOB_NAME=${JOB_NAME:-OpenJDK}
 NUM_PROCESSORS=${NUM_PROCESSORS:-$(getconf _NPROCESSORS_ONLN)}
+TMP_DIR=$(dirname $(mktemp -u))
+OPENJDK_DIR="$WORKING_DIR/$OPENJDK_REPO_NAME"
+TARGET_DIR="$WORKING_DIR"
+
+determineBuildProperties
```

Comment:
Still getting this https://github.com/AdoptOpenJDK/openjdk-build/pull/82#issuecomment-296641587, but that could be due to the above things I have pointed out.